website/server/controllers/api-v3/groups.js

```diff
@@ -166,6 +166,12 @@ api.getGroup = {
     }
 
     group = Group.toJSONCleanChat(group, user);
+
+    if (group.purchased.plan.customerId) group.purchased.active = true;
```

Comment:
group.purchased.active is defined as a field, why it's necessary to set it manually?